### ISSUE TITLE
SSD: Single Shot MultiBox Detector

### ISSUE THREAD

**Issue by joyhuang9473** [number9473/nn-algorithm#143]
# SSD: Single Shot MultiBox Detector #

- Author: Wei Liu, Dragomir Anguelov, Dumitru Erhan, Christian Szegedy, Scott Reed, Cheng-Yang Fu, Alexander C. Berg
- Origin: https://arxiv.org/abs/1512.02325
- Related:
  - http://www.cs.unc.edu/~wliu/papers/ssd_eccv2016_slide.pdf
  - https://github.com/amdegroot/ssd.pytorch
  - https://github.com/tensorflow/models/tree/master/object_detection
  - https://github.com/weiliu89/caffe/tree/ssd
  - https://github.com/balancap/SSD-Tensorflow
  - https://github.com/pby5/ssd_torch
  - mobilenet-based SSD caffemodel https://github.com/chuanqi305/MobileNet-SSD

**Comment by joyhuang9473** [comment]
How to train your own Object Detector with TensorFlow’s Object Detector API

https://medium.com/towards-data-science/how-to-train-your-own-object-detector-with-tensorflows-object-detector-api-bec72ecfe1d9

**Comment by joyhuang9473** [comment]
Tensorflow Object Detection API

https://github.com/tensorflow/models/tree/master/object_detection

**Comment by joyhuang9473** [comment]
SSD论文阅读（Wei Liu——【ECCV2016】SSD Single Shot MultiBox Detector）

https://www.bbsmax.com/A/gGdX0Vepd4/

**Comment by joyhuang9473** [comment]
SSD：Single Shot MultiBox Detector代码解读

http://blog.csdn.net/u011956147/article/details/73028773

**Comment by joyhuang9473** [comment]
Train SSD on custom dataset

https://github.com/weiliu89/caffe/wiki/Train-SSD-on-custom-dataset

**Comment by joyhuang9473** [comment]
用SSD的物体检测网络，它的label是怎么写入lmdb的

http://caffecn.cn/?/question/831

**Comment by joyhuang9473** [comment]
SSD: Signle Shot Detector 用于自然场景文字检测
- http://lib.csdn.net/article/deeplearning/50133
- https://github.com/chenxinpeng/SSD_scene_text_detection

**Comment by joyhuang9473** [comment]
Ubuntu上用caffe的SSD方法训练umdfaces数据集

http://blog.csdn.net/u013738531/article/details/61934587

**Comment by joyhuang9473** [comment]
shufflenet-based SSD caffemodel:

- src: https://bitbucket.org/ananddb90/shuffle_net/src
- issue: https://github.com/farmingyard/ShuffleNet/issues/1#issuecomment-324854508
- related: #130 

@ananddb90 @farmingyard

**Comment by joyhuang9473** [comment]
follow the ncnn's issue "Support which detection network model,Faster-RCNN or SSD or YOLO?" https://github.com/Tencent/ncnn/issues/152#issuecomment-334775869

**Comment by joyhuang9473** [comment]
mobilenet-based SSD caffemodel:

https://github.com/chuanqi305/MobileNet-SSD

#73

**Comment by dlyldxwl** [comment]
Thanks you for your job ! I'm interested in shufflenet-ssd，but the link of shufflenet-ssd caffemodel is not effective now. Can you share with me the caffemodel and train or deploy prototxt? My email is 844518133@qq.com.
Thanks again !

**Comment by 408550969** [comment]
楼主你好，请问 shufflenet-based SSD 在VOC上的精度能达到多少？
